src/openqaoa-braket/openqaoa_braket/backends/devices.py

```diff
@@ -145,4 +145,4 @@ def _check_provider_connection(self) -> bool:
             return False
 
     def connectivity(self) -> List[List[int]]:
-        return self.backend_device.topology_graph.edges
+        return list(self.backend_device.topology_graph.edges)
```

Comment:
This returns a list of tuples instead of a list of list. Should the output typings be updated or the tuple recasted into lists?

Reply:
Fixed.